config/routes.rb

```diff
@@ -306,6 +306,7 @@
       resources :connected_applications, only: %i[index destroy]
       resource :valid_va_file_number, only: %i[show]
       resources :payment_history, only: %i[index]
+      resource :military_occupations, only: :show, defaults: { format: :json }
```

Comment:
`defaults: { format: :json }` may be redundant, since this is declared on the `v0` namespace: 

```
namespace :v0, defaults: { format: 'json' } do
```